templates/circle.yml.erb

```diff
@@ -1,3 +1,9 @@
+checkout:
+  post:
+    - cp .sample.env .env
+database:
+  override:
+    - bundle exec rake db:setup
```

Comment:
`setup` script also runs seeds. Do we ever need to run seeds before test runs? I am ok with leaving DB setup to Circle CI's default.